src/clangd-context.ts

```diff
@@ -56,19 +56,28 @@ class EnableEditsNearCursorFeature implements vscodelc.StaticFeature {
   dispose() {}
 }
 
+export async function createContext(globalStoragePath: string,
```

Comment:
minor: You could make this a static async function on the `ClangdContext` class and make the constructor on `ClangdContext` private to force usage through this static factory method.

Reply:
Done; thanks for the suggestion.